content/25.app.vax-novel.md

```diff
@@ -176,14 +105,13 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Janssen’s JNJ-78436735
 
 The Johnson and Johnson (J&amp;J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&amp;J, was conducted in collaboration with and funded by “Operation Warp Speed” [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use; @doi:10.1056/NEJMoa2034201;@doi:10.1038/s41541-020-00243-x].
-The vaccine was developed using Janssen’s AdVac^®^ and PER.C6 platforms that were previously utilized to develop the European Commission-approved Ebola vaccine (Ad26 ZEBOV and MVN-BN-Filo) and their Zika, respiratory syncytial (RSV), and HIV investigational vaccine candidates [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
+The vaccine was developed using Janssen’s AdVac^®^ and PER.C6 platforms that were previously utilized to develop the European Commission-approved Ebola vaccine (Ad26 ZEBOV and MVN-BN-Filo) and their Zika, respiratory syncytial virus, and human immunodeficiency virus investigational vaccine candidates [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
 
 The development of a single-dose vaccine was desired by J&amp;J from the outset, with global deployment being a key priority [@doi:10.1038/s41586-020-2607-z].
-Using their AdVac^®^ technology, the vaccine can remain stable for up to two years between -15℃ and -25℃ and at least three months at 2-8℃ [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
+Using their AdVac^®^ technology, the vaccine can remain stable for up to two years between -15 and -25&#8451;C and at least three months at 2 to 8&#8451;C [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].<!--Confirm symbol is rendering-->
```

Comment:
```suggestion
Using their AdVac^®^ technology, the vaccine can remain stable for up to two years between -15 and -25&#8451; and at least three months at 2 to 8&#8451; [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
```
Don't need the symbol and the C. Here is the HTML artifact:


![image](https://user-images.githubusercontent.com/3532898/188271265-61d034c9-6054-4727-b7e6-cf8f79768131.png)